README.md

```diff
@@ -27,7 +176,9 @@ $ npm install loopback-connector-cloudant --save
 
 The `--save` option adds the dependency to the application’s `package.json` file.
 
-## Configuration
+# Config
+
+## generate datasource
```

Comment:
generate datasource --> Generate DataSource

In general, the title should have the proper capitalization of the first character of the word.